docs/contributing/Building, Debugging, and Testing on Windows.md

```diff
@@ -13,6 +13,14 @@
     - In the resulting window, press the 'Modify' button
     - Check the "Visual Studio Extensibility Tools Update 3" item and press the 'Next' button
     - Press the 'Update' button
+3. Install an insider VSIX from our [nightly MyGet feed](https://dotnet.myget.org/feed/roslyn/package/vsix/eb2680f2-4e63-44a8-adf6-2e667d9f689c). Note that these builds are uploaded without validation, so you may need to try to find one that works for you.
```

Comment:
I would point at something that we at least know to be decent (the most recent version inserted into the RC branch or something)?